src/MICore/CommandFactories/gdb.cs

```diff
@@ -258,5 +263,17 @@ private static string TypeBySize(uint size)
             }
             return TargetArchitecture.Unknown;
         }
+
+        public override async Task Signal(string sig)
```

Comment:
Ok. I can certainly rename this method, but notice that I am not using this command to cause a break, but to deliver specific signals to the process. The process may or may not break on those signals.